units/stats_task_end.go

```diff
@@ -116,15 +116,14 @@ func (j *collectTaskEndDataJob) Run(ctx context.Context) {
 	ctx, cancel = context.WithCancel(ctx)
 	defer cancel()
 
-	mgrOpts := cloud.ManagerOpts{
-		Provider: j.host.Provider,
-		Region:   cloud.GetRegion(j.host.Distro),
+	catcher := grip.NewBasicCatcher()
```

Comment:
I think you can just do j.AddErrors() and j.HasErrors() and don't need the extra catcher here.

Reply:
done